testsuite/smoke-test/src/test_utils.rs

```diff
@@ -57,6 +58,31 @@ pub async fn transfer_coins(
     txn
 }
 
+pub async fn transfer_and_reconfig(
+    client: &RestClient,
+    transaction_factory: &TransactionFactory,
+    root_account: &mut LocalAccount,
+    sender: &mut LocalAccount,
+    receiver: &LocalAccount,
+    num_transfers: usize,
+) {
+    for _ in 0..num_transfers {
+        // Reconfigurations have a 20% chance of being executed
+        if random::<u16>() % 5 == 0 {
```

Comment:
I wonder if you should make this have a fixed seed so it's not completely random

Reply:
Yeah, these types of non-determinism exist in a bunch of places. Once we get the MVP done, we can see if it makes sense to invest time in fixing them.